code/src/WijmoProvider/Columns/DateColumn.ts

```diff
@@ -49,7 +49,7 @@ namespace WijmoProvider.Column {
 
         public build(): void {
             //Setting date format
-            this.config.format = GridAPI.dateFormat;
+            this.config.format = this.editorConfig.format || GridAPI.dateFormat;
```

Comment:
Is this the only way to get the dateFormat? We should avoid accessing the GridAPI on the wijmoProvider

Reply:
I agree with Tiago. Also these changes are they anyhow typed in OS side? as in, can we have them typed somehow (e.g. enum)?